seth/keyfile.go

```diff
@@ -13,22 +13,27 @@ import (
 )
 
 // NewAddress creates a new address
-func NewAddress() (string, string, error) {
-	privateKey, err := crypto.GenerateKey()
+func NewAddress() (address, privateKey string, err error) {
```

Comment:
Let's stick with one way of writing return params, let's use anonymous since most of our code use them this way, easier to read